thirdparty/GL/GL.hpp

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include <common/Utils.hpp> // it includes GL/gl.h
+#include <compat/PlatformDefinitions.h>
```

Comment:
I think this shouldn't be in angled quotes

Reply:
Well you can see common/Utils.cpp included with <>. We ought to standardize includes from local to either "" or <>, but this can be done later.

Reply:
Then why the hell is the thing above it not in quotes?
I was just following what the "Utils.hpp" include did. Is there a reason it was done that way? Who knows, no one left comments...

Reply:
It's fine for now, we'll fix it later.

Reply:
> Well you can see `common/Utils.hpp` included with <>. We ought to standardize includes from local to either "" or <>, but this can be done later.

100% they should be done using quotes and not <>'s. <>'s are supposed to be used for includes from either libraries or standard headers.